contributor-guide/modules/ROOT/pages/tweeting.adoc

```diff
@@ -0,0 +1,73 @@
+= Tweeting
+:navtitle: Tweeting
+
+
+The collection is represented on social media by the https://twitter.com/boost_libraries[@Boost_Libraries]
```

Comment:
Should these external links open in other tabs?

IIRC the syntax is something like:

```
https://chat.asciidoc.org[Discuss AsciiDoc,role=resource,window=_blank]
```